src/orchard/namespace.clj

```diff
@@ -127,3 +127,8 @@
               (first paths)
               (recur (rest paths)))))))
     (catch Throwable _ nil)))
+
+(defn has-tests?
+  "Return a truthy value if the namespace has any `:test` metadata."
```

Comment:
Perhaps "namespace vars have..." would make it more clear what this actually does. :-)